husarnet/husarnet_manager.cpp

```diff
@@ -317,6 +318,40 @@ HusarnetManager::HusarnetManager()
   Privileged::init();
 }
 
+void HusarnetManager::readLegacyConfig()
+{
+  const std::string legacyConfigPath = Privileged::getLegacyConfigPath();
+  if(Port::isFile(legacyConfigPath)) {
```

Comment:
Nit pick/style suggestion: If you have a list of checks that are enclosing most of the code in a function consider changing them into negated checks with an error message/response and a return inside. This way the bulk of the code will be indented one level less and possibly a little bit more readable (while reading it you won't have to "go one level deeper"). In this case it'd make this code something like: if that file is **not** present - return.